src/View/TwigView.php

```diff
@@ -198,6 +198,7 @@ protected function initializeExtensions(): void
         $this->twig->addExtension(new Extension\StringsExtension());
         $this->twig->addExtension(new Extension\TimeExtension());
         $this->twig->addExtension(new Extension\UtilsExtension());
+        $this->twig->addExtension(new Extension\ViewExtension());
```

Comment:
Should this be `ViewHelperExtension`?

Reply:
I intended to add more view related function to the extension hence the name. For e.g. a `fetch` function so that we can use `{{ fetch('block') }}` instead of `{{ _view.fetch('block')|raw }}`.

Reply:
I was thinking it might be cleaner to add functions/filters for TwigView itself directly in TwigView.

Seems odd to allow disabling TwigView-specific functions. But I won't argue if this feels correct to you.

Reply:
> I was thinking it might be cleaner to add functions/filters for TwigView itself directly in TwigView.

That would would make the TwigView class too bloated IMO.

We aren't actually adding any new functionality to the View classes. This extension class just adds an additional layer of abstraction for accessing the View class methods/properties.

Reply:
ok. We can add the element and cell functions to it.